kotlin-generator/src/main/scala/models/generator/kotlin/KotlinGenerator.scala

```diff
@@ -43,10 +47,24 @@ class KotlinGenerator
       val allEnumValues = enum.values ++ Seq(io.apibuilder.spec.v0.models.EnumValue(undefinedEnumName, Some(undefinedEnumName)))
 
       allEnumValues.foreach(value => {
-        val annotation = AnnotationSpec.builder(classOf[JsonProperty]).addMember("value", "\"" + value.name + "\"")
-        builder.addEnumConstant(toEnumName(value.name), TypeSpec.anonymousClassBuilder("").addAnnotation(annotation.build()).build())
+        val annotation = AnnotationSpec.builder(classOf[JsonProperty]).addMember("\"" + value.name + "\"")
+        builder.addEnumConstant(toEnumName(value.name), TypeSpec.anonymousClassBuilder(""""""" +  s"${value.name}" + """"""").addAnnotation(annotation.build()).build())
```

Comment:
I'd argue `s"\"${value.name}\""` or `"\"" + value.name + "\""` is more readable than `""""""" +  s"${value.name}" + """""""`

Reply:
changed to "\"" + value.name + "\""